tests/class.phpt

```diff
@@ -19,6 +19,7 @@ echo ast_dump(ast\parse_code($code, $version=60));
 
 ?>
 --EXPECTF--
+Deprecated: ast\parse_code(): Version 60 is deprecated in %s.php on line 15
```

Comment:
Why does this one not update the used version?

Reply:
Some tests of the behavior in older ast declaration versions were deliberately left in to prevent accidental regressions. The resulting AST was different when I switched these to tests of AST version 70 (e.g. AST_PROP_GROUP, removal of `'uses'` child node in newer ast versions for some node kinds, etc).